ansible/roles/ocp4-workload-dil-agile-integration/tasks/user_sso_realm.yaml

```diff
@@ -1,8 +1,19 @@
 ---
 
-- name: Create  user{{__user_number}} realm
-  k8s:
-    state: present
-    resource_definition: "{{ lookup('template', 'sso-user-realm.yaml.j2') }}"
-  vars:
-    user: 'user{{__user_number}}'
+- set_fact: current_user='user{{__user_number}}'
+
+
+
+- name: Create {{ current_user }} Realm
+  uri: 
+    url: 'https://keycloak-{{sso_project}}.{{route_subdomain}}/auth/admin/realms'
+    method: POST
+    body_format: json
+    headers:
+      Content-Type: "application/json"
+      Authorization: "Bearer {{TKN}}"
+    body:  "{{ lookup('file', 'sso-user-realm.json') | replace('%user%', current_user) | from_json }}"
```

Comment:
Why use `replace` instead of just passing a variable and use  `template` instead of `file` ?